fplc_method_validator.py

```
@@ -0,0 +1,497 @@
+import json
+import logging
+from dataclasses import dataclass, field
```

Ruff (0.12.2)

`dataclasses.field` imported but unused

Remove unused import: `dataclasses.field`

(F401)